.storybook/preview-body.html

```diff
@@ -12,6 +12,10 @@
 		font-size: 12px;
 	}
 
+	body {
+		min-height: 100vh;
+	}
```

Comment:
This fixes the custom background when changing it. It wasn't covering the full height.